BUILD

```diff
@@ -67,14 +67,28 @@ container_bundle(
         "gcr.io/{PROJECT_ID}/python2.7:latest": "//experimental/python2.7:python27",
         "gcr.io/{PROJECT_ID}/python2.7:debug": "//experimental/python2.7:debug",
-        "gcr.io/{PROJECT_ID}/nodejs:latest": "//experimental/nodejs:nodejs10",
-        "gcr.io/{PROJECT_ID}/nodejs:10": "//experimental/nodejs:nodejs10",
-        "gcr.io/{PROJECT_ID}/nodejs:12": "//experimental/nodejs:nodejs12",
-        "gcr.io/{PROJECT_ID}/nodejs:14": "//experimental/nodejs:nodejs14",
-        "gcr.io/{PROJECT_ID}/nodejs:debug": "//experimental/nodejs:nodejs10_debug",
-        "gcr.io/{PROJECT_ID}/nodejs:10-debug": "//experimental/nodejs:nodejs10_debug",
-        "gcr.io/{PROJECT_ID}/nodejs:12-debug": "//experimental/nodejs:nodejs12_debug",
-        "gcr.io/{PROJECT_ID}/nodejs:14-debug": "//experimental/nodejs:nodejs14_debug",
+        "gcr.io/{PROJECT_ID}/nodejs:latest": "//experimental/nodejs:nodejs10_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:10": "//experimental/nodejs:nodejs10_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:12": "//experimental/nodejs:nodejs12_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:14": "//experimental/nodejs:nodejs14_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:debug": "//experimental/nodejs:nodejs10_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:10-debug": "//experimental/nodejs:nodejs10_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:12-debug": "//experimental/nodejs:nodejs12_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:14-debug": "//experimental/nodejs:nodejs14_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:10": "//experimental/nodejs:nodejs10_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:12": "//experimental/nodejs:nodejs12_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:14": "//experimental/nodejs:nodejs14_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:debug": "//experimental/nodejs:nodejs10_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:10-debug": "//experimental/nodejs:nodejs10_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:12-debug": "//experimental/nodejs:nodejs12_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:14-debug": "//experimental/nodejs:nodejs14_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian10:10": "//experimental/nodejs:nodejs10_debian10",
+        "gcr.io/{PROJECT_ID}/nodejs-debian10:12": "//experimental/nodejs:nodejs12_debian10",
+        "gcr.io/{PROJECT_ID}/nodejs-debian10:14": "//experimental/nodejs:nodejs14_debian10",
+        "gcr.io/{PROJECT_ID}/nodejs-debian10:debug": "//experimental/nodejs:nodejs10_debug_debian10",
+        "gcr.io/{PROJECT_ID}/nodejs-debian10:10-debug": "//experimental/nodejs:nodejs10_debug_debian10",
+        "gcr.io/{PROJECT_ID}/nodejs-debian10:12-debug": "//experimental/nodejs:nodejs12_debug_debian10",
+        "gcr.io/{PROJECT_ID}/nodejs-debian10:14-debug": "//experimental/nodejs:nodejs14_debug_debian10",
```

Comment:
Thanks for updating this. However, the actual file that publishes images is `cloudbuild.yaml`. Could you update that file as well?

Reply:
done

Reply:
I pointed all "latest" to node 14 since the official node docker node:latest is also version 14

Reply:
I don't know anything about Node.js. If 14 is backward-compatible and can seamlessly replace 10, I think it's OK?

Reply:
While mostly ok, I don't think it is guaranteed. However isn't that what you would expect when using a "latest" image. I feel like it would be strange to use node:latest and get node 10. See https://hub.docker.com/_/node , the official node image does the same thing. 
![image](https://user-images.githubusercontent.com/2118704/91937874-2b8b9f00-ecf3-11ea-8f94-b7f5c2ede674.png)

Reply:
Makes sense!